### DISCUSSION TITLE
URBANopt and OpenModelica issues

### DISCUSSION THREAD

**Discussion by bvidalEKT** [urbanopt/urbanopt-cli#415] [Q&A]
Hey there! I'm  trying to simulate a URBANopt model in OpenModelica.

I'm using URBANopt to get load profiles of buildings in the network. I plan on simulating the network in OpenModelica using the District Energy System (DES) Workflow.

I managed to create the Modelica model through the Windows console using "uo_des create-model" after installing GMT and BML 9.1.0. But when we tried to simulate the model in OpenModelica 1.20, we couldn't get it to work.

I also downloaded the example repository "urbanopt/geojason-modelica-translator-examples" and ran "poetry run pytest".

When we opened the "Package.mo" file of the three examples ("geojson_8_buildings", "spawn_geojson", or "time_series_5g"), we got a "translation error" in OpenModelica. The error: 'Class time series 5g.Loads.BSa99ec.BuildingTimeSeries not found in scope building'

I'd really appreciate any advice or insights on how to resolve this issue. Thanks

**Comment by kflemin** [comment]
We have recently released newer versions of URBANopt which addresses many prior issues. If you are still experiencing issues related to this after upgrading, please let us know.
